frontend/src/views/Bridge.vue

```diff
@@ -210,12 +249,29 @@
                   {{supportedChains[supportedChainIds.indexOf(character['2'])]}}
                 </option>
               </select>
-              <div class="p-2 text-center">
+              <div class="mt-2 text-center">
                 <b-button :disabled="!characterIdToWithdraw" variant="primary"
-                  @click="withdrawBridge(characterIdToWithdraw)" class="gtag-link-others"
+                  @click="withdrawBridge(characterIdToWithdraw)" class="gtag-link-others withdrawBtn"
                   tagname="click_transfer_bridge">{{$t('bridge.withdrawCharacter')}}</b-button>
               </div>
             </div>
+            <div v-if="incomingShields.length !== 0" class="p-4 w-20">
```

Comment:
Missing translations in this div

Reply:
resolved in 9153fdf622c7d8f6f35ae769c2caa66d20392fda